wazimap_ng/boundaries/models.py

```diff
@@ -16,4 +18,13 @@ class GeographyBoundary(models.Model):
     class Meta:
         indexes = [models.Index(fields=["code"])]
 
-
+	def save(self, *args, **kwargs):
+		"""
+		On save update Locations data
+		"""
+		super().save(*args, **kwargs)
+		async_task(
+			"wazimap_ng.boundaries.tasks.update_location_geo_levels_data",
+			self,
+			task_name=f"Updating geo boundary data: {self.name}",
+		)
```

Comment:
Can we move this to a signal instead of the save method? The boundary model shouldn't be so tightly coupled the location. I think the signal should live in the points app rather than the boundaries app